index.html

```diff
@@ -289,5 +289,61 @@ <h1 class="title">Skills</h1>
 
       <!-- ^^^^^^^^^^^^^ Category skills in about section ^^^^^^^^ -->
     </section>
+
+    <!-- -------- contact-section ---------- -->
+    <section class="contact-section">
+      <div class="contact">
+        <div class="contact__header">
+          <h1 class="header__title">Contact Me</h1>
+          <p class="header__text">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+
+        <form

```

Comment:
The form does not have any styling and does not follow the design. Kindly follow the Figma design which includes color, background images, button hover effects, etc.

Yours:
![yours-form](https://user-images.githubusercontent.com/35504149/171192856-3adef4c3-a8b4-43c1-a4c9-e5a1abf2a5fb.png)

Design:
![form1](https://user-images.githubusercontent.com/35504149/171192960-e2056b20-888d-46d1-b45b-c279a9b84992.png)
![form2](https://user-images.githubusercontent.com/35504149/171193000-c38e462b-0d12-45d5-aed0-61d14135622e.png)

Reply:
- [ opional] next time please try to start styling normally with the mobile version and only include media query when switching to the desktop version.